unstructured/staging/base.py

```diff
@@ -337,3 +338,92 @@ def filter_element_types(
         return filtered_elements
 
     return elements
+
+
+def convert_to_coco(
+    elements: List[Element],
+    dataset_description: str = f"Unstructured COCO Dataset {datetime.now().strftime('%Y-%m-%d')}",
+    dataset_version: str = "1.0",
+    contributors: List = ["Unstructured Developers"],
```

Comment:
It might be safer to make this a `Tuple[str]` instead. Mutable default values can result in some weird behavior, see [here](https://unstructuredai.slack.com/archives/C0416NK4BLM/p1687997373994349). Since the list doesn't get changed in the function the weird behavior won't occur here, but making it a `Tuple` doesn't leave a landmine.